Formula/i2pd.rb

```diff
@@ -0,0 +1,23 @@
+class I2pd < Formula
+  desc "Full-featured C++ implementation of I2P client."
+  homepage "http://i2pd.website/"
+  head "https://github.com/PurpleI2P/i2pd.git"
+
+  stable do
+    url "https://github.com/PurpleI2P/i2pd/archive/2.8.0.tar.gz"
+    sha256 "3f3f538b3c5b3095021311b434456c42d6f3b5e43dc7972c263b10ef179ba063"
+  end
+
+  depends_on "libressl"
+  depends_on "boost"
+
+  def install
+    system "make", "HOMEBREW=1"
+
+    bin.install "i2pd"
```

Comment:
@dunn 
I've run through Make file and the only option to build sources on OSX is set `make HOMEBREW=1`. On this scenario sources builds for OSX and binary file works perfectly. 
Although I've trying to build sources with `make install` and compilation fails with message "No target for 'install'", I suppose that author want to separate source building process and create this rule. So that is really a problem? Because if it is a problem I'll try to suggest for author another scenario.